cernopendata/modules/records/serializers/basic_json.py

```diff
@@ -74,6 +74,15 @@ def serialize_search(
                     agg.pop('filtered')
 
         aggregations = aggregations[0]
+
+        # Remove empty buckets in event_numbers facet
```

Comment:
Hi @sksDonni, thanks for the contribution.

FYI the `master` branch build was broken, and the CI was not passing for your pull request, so I fixed the build and rebased, fixing the code style cosmetics.

If you would like to proceed with merging:

(1) Since this is your first time contributing to the CERN Open Data portal code base, may I ask you to reply to this comment to say that you explicitly agree donating this and any future code contributions to our repositories under the corresponding FLOSS licenses i.e. the GNU General Public License? 

(2) Your commit is authored as:

```
Author: Cloud User <centos@ip-172-26-5-255.ap-south-1.compute.internal>
```

It would be good to update it with your real name and email address.

Reply:
Thank you so much @tiborsimko. I agree that I am donating the code patch for cern opendata. 

I faced the same issue while building my local development and thought that there were some support issues as I use a macbook running on apple silicon. I therefore used a centos cloud instance to push the code.

I will add my token for future commits. Thank you once again.

Reply:
Name: Suchith Krishna S Donni
Email: sksdonni18@gmail.com

Reply:
Thanks, amended, and I have added you to the list of contributors in the AUTHORS file as well.

Reply:
Thank you! could not be more happier!